sw/device/lib/dif/dif_spi_device.h

```diff
@@ -22,18 +22,32 @@ extern "C" {
 #endif  // __cplusplus
 
 /**
- * A signal edge type: positive or negative.
+ * Clock polarity type
  */
-typedef enum dif_spi_device_edge {
+typedef enum dif_spi_device_clock_polarity {
   /**
-   * Represents a positive edge (i.e., from low to high).
+   * Positive polarity (CPOL=0)
```

Comment:
The contents of the parens are code, so they should be backticked. E.g., `` `CPOL = 0` ``.